app/searches/locations_search.rb

```diff
@@ -9,9 +9,14 @@ class LocationsSearch
   attribute :org_name, type: String
   attribute :category_ids, type: Array
   attribute :tags, type: String
+  attribute :archived_at, type: Date
+  attribute :archived, type: Boolean
+  attribute :services, type: String
```

Comment:
Why do we need this services string attribute?

Reply:
This was left over from a elasticsearch query I was writing before. Removing now.